cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -13,7 +13,6 @@ spec:
         app: csi-cephfsplugin
     spec:
       serviceAccount: rook-csi-cephfs-plugin-sa
-      hostNetwork: true
```

Comment:
why this is removed?

Reply:
This flag causes the plugin pods to share the ip of the node they run on so when both the rbd plugin and the cephfs plugin are running on the same node the liveness sidecars try to open an endpoint but one will fail because the other is already using that port on the ip they share. Removing this causes all pods to get there own ip resolving this problem. Removing it caused no issues in my testing which makes me question why it was there in the first place?